### ISSUE TITLE
Proposals: 2 const. proposal with manual trigger period has visibility issues about voting period + execution timeline

### ISSUE THREAD

**Issue by mochet** [Joystream/pioneer#4773]
# Problem
![image](https://github.com/Joystream/pioneer/assets/50830161/ba75223c-6109-4e46-b13c-cb33cee0f76c)

Link to proposal: https://pioneerapp.xyz/#/proposals/preview/787

* The time left seems to be indicating how long until the grace period would start supposing that it is actually approved
* Typically the time left is used to indicate how long the current council has to vote on the proposal, this could be misleading and make the current council believe they have ~12 days left to vote when it is ~6 days

**Comment by thesan** [comment]
I agree this is misleading. Ultimately this should be solved by some greatly improved design as suggested in #4269.

In the meantime "Time left" should indicate clearly what it's about. IMO this should be different for every stage:
- Deciding stage: "Time left to vote"
- Dormant stage: "Time until deciding stage"
- Gracing stage: "Time until execution"

In addition during the Deciding and Dormant stages I think that proposal without an exact execution block should have an additional statistic showing the earliest possible execution and the latest possible execution.

**Comment by thesan** [comment]
@mochet I'm not sure if it's what you mean, but there is a statistic that shows that the proposal has an exact execution block: 
![image](https://github.com/Joystream/pioneer/assets/6571453/c21f8381-9362-4d5e-9c39-f9adbf323e72)

Also could you clarify what would be a harmful consequence of a council voting on a proposal with a trigger without realizing it ?
